vars/main.yml

```diff
@@ -14,15 +14,21 @@ k8s:
 core:
   standalone: "true"
   data_iface: ens18
+  # If upf.mode set to 'dpdk' then use below file for 'values_file',
+  # "deps/5gc/roles/core/templates/sdcore-5g-sriov-values.yaml"
```

Comment:
What about moving this comment to the `mode` section below
```suggestion
```

Reply:
done